openpype/hosts/houdini/plugins/publish/validate_vdb_output_node.py

```diff
@@ -55,6 +55,8 @@ def get_geometry_at_frame(sop_node, frame, force=True):
     """Return geometry at frame but force a cooked value."""
     with update_mode_context(hou.updateMode.AutoUpdate):
         sop_node.cook(force=force, frame_range=(frame, frame))
+        if not hasattr(sop_node, "geometry"):
+            return
```

Comment:
Can we do this check outside of the with statement? That way we don't force a cook and change update states for no reason.

Reply:
good observation! 
thank you